comp/core/workloadmeta/collectors/internal/containerd/container_builder.go

```diff
@@ -57,6 +57,16 @@ func buildWorkloadMetaContainer(namespace string, container containerd.Container
 		}
 	}
 
+	// Get Container PID
+	var pid int
+	task, err := container.Task(ctx, nil)
+	if err == nil {
+		pid = int(task.Pid())
```

Comment:
what is the origin type of the Pid() ? uint?

Reply:
yes